images/rootfs-kubevirt.yml.in.yq

```diff
@@ -0,0 +1 @@
+.services |= .+ [{"name": "eve-k3s","image": "K3S_TAG", "cgroupsPath": "/eve/services/eve-k3s", "oomScoreAdj": -999}]
```

Comment:
What about calling it just `k3s`? None of the existing services uses "eve-" prefix. This is upstream k3s that we are installing, not a special patched version.
Also, in order to enter the container, we will be able to write shorter `eve enter k3s` as opposed to `eve enter eve-k3s`.

Reply:
@milan-zededa the naming of eve-k3s was intentional. We do have multiple k3s env in our eco system. Like k3s runtime we support and also customer can have their own k3s deployed in a VM. To avoid any confusions I wanted to explicitly call it eve-k3s.   Also, in future this eve-k3s container can be deployed as an app itself if customer wants (since its self contained). 

So basically trying to avoid confusion and be explicit.

Reply:
@zedi-pramodh but a user would not run a Linuxkit container inside EVE. Customers are free to name their containers debug, wwan, or k3s even though EVE-OS contains Linuxkit containers with those names.

Reply:
Ok then how about we just call our container Kubernetes instead of eve-k3s.

Reply:
Discussed with @naiming-zededa too and decision is to call our container Kube, to keep it short and simple. If there are no further comments I will go ahead and make that change.

Reply:
@zedi-pramodh I'm OK with Kube, you can go ahead and make that change.